web/modules/custom/joinup_federation/src/Plugin/pipeline/Step/BrokenReferences.php

```diff
@@ -35,7 +35,7 @@ class BrokenReferences extends JoinupFederationStepPluginBase implements Pipelin
    *
    * @var int
    */
-  const BATCH_SIZE = 10;
```

Comment:
Also I'm not sure about the gain of lowering the batch sizes... for all plugins.

Reply:
Me neither... I think we have a more complex issue with cloudfront :(